Tests/Charts/EquatableTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  EquatableTests.swift
```

Comment:
I made the same template as yours for combined chart test. Shall we include the license info just like source code template, or test files is fine with default Xcode template?

Reply:
I guess we should be putting in the license info to be consistent. Do we keep it the same, but whose name do we use?

Reply:
I guess just the same template as source code. I don't know the rule of names. 
Personally I just use `Copyright 2015 Daniel Cohen Gindi & Philipp Jahoda`, and I don't need to specify who created the file. 
I think this is a good question, as I saw some files with yours template:
```
//
//  DemoBaseViewController.swift
//  ChartsDemo-iOS
//
//  Created by Jacob Christie on 2017-07-03.
//  Copyright © 2017 jc. All rights reserved.
//
```
I guess we need an unified one. @petester42 @danielgindi @PhilJay insights?

Reply:
Yea, I just use the Xcode default.